.github/workflows/coveritup.yml

```diff
@@ -55,6 +56,14 @@ jobs:
           echo SCORE=$(($SECONDS-BUILD_START)) >> "$GITHUB_ENV"
         shell: bash
 
+      - name: Lint Errors
```

Comment:
ah yes and I removed the separate workflow file. 
Once we have zero counts on the lint, will update this CI to exit 1 upon any errors.